### ISSUE TITLE
Exceptions on startup 4.2.65

### ISSUE THREAD

**Issue by SlimeDog** [lucko/LuckPerms#1142]
Spigot 1.13 latest
LuckPerms 4.2.65

```
[15:24:28] [Server thread/INFO]: [LuckPerms] Loading LuckPerms v4.2.65
[15:24:29] [Server thread/INFO]: [LuckPerms] Successfully downloaded 'text-1.11-1.6.1.jar' with matching checksum: uJC1/KZz/98KdOfAYjBEAR4jLGS0SOY03wUrDLTj3sg=
[15:24:29] [Server thread/INFO]: [LuckPerms] Successfully downloaded 'event-2.0.1.jar' with matching checksum: 4XVmowvKHCmO38ZO2jeZKh/mD00xBg73zau79s4kTe4=
[15:24:29] [Server thread/INFO]: [LuckPerms] Attempting to apply relocations to text-1.11-1.6.1.jar...
[15:24:29] [Server thread/ERROR]: [LuckPerms] Unable to remap the source file 'TEXT'.
[15:24:29] [Server thread/WARN]: java.lang.reflect.InvocationTargetException
[15:24:29] [Server thread/WARN]:        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[15:24:29] [Server thread/WARN]:        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[15:24:29] [Server thread/WARN]:        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[15:24:29] [Server thread/WARN]:        at java.lang.reflect.Method.invoke(Method.java:498)
[15:24:29] [Server thread/WARN]:        at me.lucko.luckperms.common.dependencies.relocation.RelocationHandler.remap(RelocationHandler.java:82)
[15:24:29] [Server thread/WARN]:        at me.lucko.luckperms.common.dependencies.DependencyManager.loadDependencies(DependencyManager.java:177)
[15:24:29] [Server thread/WARN]:        at me.lucko.luckperms.common.plugin.AbstractLuckPermsPlugin.load(AbstractLuckPermsPlugin.java:90)
[15:24:29] [Server thread/WARN]:        at me.lucko.luckperms.bukkit.LPBukkitBootstrap.onLoad(LPBukkitBootstrap.java:130)
[15:24:29] [Server thread/WARN]:        at org.bukkit.craftbukkit.v1_13_R1.CraftServer.loadPlugins(CraftServer.java:319)
[15:24:29] [Server thread/WARN]:        at net.minecraft.server.v1_13_R1.DedicatedServer.init(DedicatedServer.java:213)
[15:24:29] [Server thread/WARN]:        at net.minecraft.server.v1_13_R1.MinecraftServer.run(MinecraftServer.java:686)
[15:24:29] [Server thread/WARN]:        at java.lang.Thread.run(Thread.java:748)
[15:24:29] [Server thread/WARN]: Caused by: java.lang.RuntimeException: Error processing class net/kyori/text/serializer/GsonComponentSerializer.class
[15:24:29] [Server thread/WARN]:        at me.lucko.jarrelocator.JarRelocator.addRelocatedClass(JarRelocator.java:162)
[15:24:29] [Server thread/WARN]:        at me.lucko.jarrelocator.JarRelocator.run(JarRelocator.java:115)
[15:24:29] [Server thread/WARN]:        ... 12 more
[15:24:29] [Server thread/WARN]: Caused by: java.lang.ArrayIndexOutOfBoundsException: 853
[15:24:29] [Server thread/WARN]:        at org.objectweb.asm.ClassReader.readLabel(ClassReader.java:2442)
[15:24:29] [Server thread/WARN]:        at org.objectweb.asm.ClassReader.createLabel(ClassReader.java:2458)
[15:24:29] [Server thread/WARN]:        at org.objectweb.asm.ClassReader.readTypeAnnotations(ClassReader.java:2523)
[15:24:29] [Server thread/WARN]:        at org.objectweb.asm.ClassReader.readCode(ClassReader.java:1688)
[15:24:29] [Server thread/WARN]:        at org.objectweb.asm.ClassReader.readMethod(ClassReader.java:1274)
[15:24:29] [Server thread/WARN]:        at org.objectweb.asm.ClassReader.accept(ClassReader.java:679)
[15:24:29] [Server thread/WARN]:        at org.objectweb.asm.ClassReader.accept(ClassReader.java:391)
[15:24:29] [Server thread/WARN]:        at me.lucko.jarrelocator.JarRelocator.addRelocatedClass(JarRelocator.java:160)
[15:24:29] [Server thread/WARN]:        ... 13 more
[15:24:29] [Server thread/INFO]: [LuckPerms] Attempting to apply relocations to event-2.0.1.jar...
```
```
[15:24:30] [Server thread/INFO]: [LuckPerms] Enabling LuckPerms v4.2.65
[15:24:30] [Server thread/ERROR]: Error occurred while enabling LuckPerms v4.2.65 (Is it up to date?)
java.lang.NoClassDefFoundError: me/lucko/luckperms/lib/text/Component
        at me.lucko.luckperms.bukkit.BukkitSenderFactory.getUuid(BukkitSenderFactory.java:70) ~[?:?]
        at me.lucko.luckperms.bukkit.BukkitSenderFactory.getUuid(BukkitSenderFactory.java:47) ~[?:?]
        at me.lucko.luckperms.common.sender.AbstractSender.<init>(AbstractSender.java:58) ~[?:?]
        at me.lucko.luckperms.common.sender.SenderFactory.wrap(SenderFactory.java:67) ~[?:?]
        at me.lucko.luckperms.bukkit.LPBukkitPlugin.getConsoleSender(LPBukkitPlugin.java:364) ~[?:?]
        at me.lucko.luckperms.common.plugin.AbstractLuckPermsPlugin.enable(AbstractLuckPermsPlugin.java:98) ~[?:?]
        at me.lucko.luckperms.bukkit.LPBukkitBootstrap.onEnable(LPBukkitBootstrap.java:152) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[spigot-1.13.jar-2018-08-09-0653:git-Spigot-417cfb3-4487c1f]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:339) [spigot-1.13.jar-2018-08-09-0653:git-Spigot-417cfb3-4487c1f]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:403) [spigot-1.13.jar-2018-08-09-0653:git-Spigot-417cfb3-4487c1f]
        at org.bukkit.craftbukkit.v1_13_R1.CraftServer.enablePlugin(CraftServer.java:425) [spigot-1.13.jar-2018-08-09-0653:git-Spigot-417cfb3-4487c1f]
        at org.bukkit.craftbukkit.v1_13_R1.CraftServer.enablePlugins(CraftServer.java:339) [spigot-1.13.jar-2018-08-09-0653:git-Spigot-417cfb3-4487c1f]
        at net.minecraft.server.v1_13_R1.DedicatedServer.init(DedicatedServer.java:214) [spigot-1.13.jar-2018-08-09-0653:git-Spigot-417cfb3-4487c1f]
        at net.minecraft.server.v1_13_R1.MinecraftServer.run(MinecraftServer.java:686) [spigot-1.13.jar-2018-08-09-0653:git-Spigot-417cfb3-4487c1f]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_171]
Caused by: java.lang.ClassNotFoundException: me.lucko.luckperms.lib.text.Component
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381) ~[?:1.8.0_171]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:154) ~[spigot-1.13.jar-2018-08-09-0653:git-Spigot-417cfb3-4487c1f]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:100) ~[spigot-1.13.jar-2018-08-09-0653:git-Spigot-417cfb3-4487c1f]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_171]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_171]
        ... 15 more
```

**Comment by lucko** [comment]
Thanks for reporting, looking into it now.

You can use the previous version for the time being, doesn't have the updated dependency version.
https://ci.lucko.me/job/LuckPerms/724/

**Comment by SlimeDog** [comment]
Confirmed fixed in LuckPerms 4.2.66
